payments/models.py

```diff
@@ -127,14 +128,21 @@ class WinterStorageProduct(AbstractPlaceProduct):
         related_name="product",
         on_delete=models.CASCADE,
     )
+    price_unit = models.CharField(
+        choices=[(PriceUnits.AMOUNT, PriceUnits.AMOUNT.label)],
+        default=PriceUnits.AMOUNT,
+        max_length=10,
+    )
```

Comment:
why is this needed, if it's already defined on the `AbstractPlaceProduct`? 🤔

Reply:
Excellent question :derp: removed now.